src/test/integration/redis/tested_versions/redis

```diff
@@ -0,0 +1 @@
+4.6.0
```

Comment:
is this a symlink, or an actual copy of the file?

Reply:
great catch!!